test/js/simple-length.js

```diff
@@ -90,4 +90,24 @@ suite('SimpleLength', function() {
     assert.strictEqual(lengthAddition.value, 5);
   });
 
+  test('convertToCalcLength method returns a CalcLength object with a single length value type when called from a SimpleLength object', function() {
+    var simpleLength = new SimpleLength(10, 'em');
+    var convertedCalcLength = simpleLength._convertToCalcLength();
+    var testCalcLength = new CalcLength({em: 10})
+    assert.isTrue(convertedCalcLength.equals(testCalcLength));
+  });
+
+  test('Test equals method of Simple length such that it will return true if two simple lengths are the same', function() {
```

Comment:
Similar to the calc length tests, how about

"Equals method should return true for equal SimpleLengths"